app/learn/tutorials/quickstart-uplink-cli/generate-access-grants-and-tokens/page.md

```diff
@@ -6,6 +6,6 @@ redirects:
 weight: 15
 ---
 
-[](docId:TbMdOGCAXNWyPpQmH6EOq)&#x20;
+[](docId:TbMdOGCAXNWyPpQmH6EOq)
 
 [](docId:OXSINcFRuVMBacPvswwNU)&#x20;
```

Comment:
Missed some. Looks like if it's at the end of the file?

Reply:
Hopefully got all the rest of them. There was one place I didn't know what to do with it.